src/main/pages/che-7/end-user-guide/assembly_defining-custom-commands-for-che-theia.adoc

```diff
@@ -18,78 +18,15 @@ summary:
 
 :context: defining-custom-commands-for-che-theia
 
-The Che-Theia IDE allows users to define custom commands in a devfile that are then available when working in a workspace.
+When starting a new project, new developers often loose time in finding the good procedure to build, run or debug a project. The result is documentation that are hard to follow and never up-to-date.
 
-The following is an example of the `commands` section of a devfile.
+On boarding new developers is one of the main Che challenges. To help the new developers, lead developers can define devfiles to create on-demand workspaces for specific projects. 
```

Comment:
The sentence

> On boarding new developers is one of the main Che challenges.

makes it sound like the documentation is saying "it's hard to start using Che"

Reply:
s/challenges/purposes/gc ?

Reply:
Hm, I don't know that "purposes" works either, since then it's implying that Che is an onboarding tool.

Reply:
Che is a good IDE for onboarding new developers in a project. Predefined commands is helping.